client/src/components/_common/Searchbar/Searchbar.jsx

```diff
@@ -137,7 +137,7 @@ const Searchbar = ({
           className={`${styles.results} ${disabled ? styles.hidden : ''}`}
           data-testid="summary-of-search-results"
         >
-          {resultCount} results in {sectionName}
+          {resultCount} results in {sectionName == '' ? dataType : sectionName}
```

Comment:
No action required. Another way to do this would be
```suggestion
          {resultCount} results in {sectionName || dataType}
```